portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,6 +31,8 @@
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Streams;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
```

Comment:
nit: imports should be ordered and grouped alphabetically, most linters will do this automatically